### PR TITLE
Authentication #536

### PR DESCRIPTION
**Pull request by ordansantos**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,7 +86,7 @@
 		<dependency>
 		    <groupId>org.apache.httpcomponents</groupId>
 		    <artifactId>httpclient</artifactId>
-		    <version>4.5.5</version>
+		    <version>4.5.3</version>
```
Comment (paulofelipefeitosa) [inline pom.xml]: Why do the version downgrade?

Comment (ordansantos) [reply pom.xml]: Sorry for that, I can't figure out why the latest version doesn't work on my workstation.

File: src/test/java/org/fogbowcloud/manager/core/services/AuthenticationServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		Integer statusResponse = HttpStatus.SC_UNAUTHORIZED;
+		Mockito.doThrow(new RuntimeException(statusResponse.toString())).when(ldapIdentityPlugin).getToken(Mockito.anyString());
+		
+		try {
+			authenticationService.authenticate(Mockito.anyString());
+			Assert.fail();
+		} catch (RuntimeException runtimeException) {
+			Assert.assertEquals(statusResponse.toString(), runtimeException.getMessage());
+		}
+
```
Comment (fgan1) [inline src/test/java/org/fogbowcloud/manager/core/services/AuthenticationServiceTest.java]: Remove this lines.

File: src/test/java/org/fogbowcloud/manager/core/services/AuthenticationServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+	public void testAuthenticateService() {
+		Token token = new Token();
+		Mockito.doReturn(token).when(ldapIdentityPlugin).getToken(Mockito.anyString());
+		Assert.assertEquals(token, authenticationService.authenticate(Mockito.anyString()));
+	}
+	
+	@Test
+	public void testInvalidAccessid() {
+		Integer statusResponse = HttpStatus.SC_UNAUTHORIZED;
+		Mockito.doThrow(new RuntimeException(statusResponse.toString())).when(ldapIdentityPlugin).getToken(Mockito.anyString());
```
Comment (fgan1) [inline src/test/java/org/fogbowcloud/manager/core/services/AuthenticationServiceTest.java]: Do you know which exceptions can be throws in the Identity Plugin ?

**Pull request merged**